src/swagger.ts

```diff
@@ -82,18 +85,32 @@ function defineRoutes(paths: OpenAPIV2.PathsObject, controllers: Controllers): R
   return router;
 }
 
-export default function swagger(swaggerFile: string, controllers: Controllers): Router {
+const defaultOptions: Options = {
+  middleware: [],
```

Comment:
preMiddleware? because it's run before the main swagger middleware?

Reply:
good point!